resources/quote.py

```diff
@@ -48,17 +52,22 @@ def post(self) -> Response:
             response, status = quote.to_json(), 201
             current_app.logger.info(f"POST Quote {quote.id}")
         except DoesNotExist:
-            response, status = objectToJson({"Error": "Author id Does Not Exist!"}), 404
-            current_app.logger.error(f"GET Quote: Author {id} not found")
-        except Exception as e:
-            response, status = objectToJson({"Error": str(e)}), 400
-            current_app.logger.error(f"POST Quote {str(e)}")
+            response, status = (
+                object_to_json({"Error": "Author id Does Not Exist!"}),
+                404,
+            )
+            current_app.logger.error("GET Quote: Author not found")
```

Comment:
Do we need this if we are going to add NEW quote anyway?

Reply:
Yes, If the author ID provided with the quote does not exist then this exception will be raised.

Reply:
Can you show me the request body for this request?